azure_storage_client.go

```diff
@@ -204,7 +204,7 @@ func (util *snowflakeAzureClient) uploadFile(
 		})
 	} else {
 		var f *os.File
-		f, err = os.OpenFile(dataFile, os.O_RDONLY, os.ModePerm)
+		f, err = os.OpenFile(dataFile, os.O_RDONLY, 0666)
```

Comment:
Can we introduce a const instead of repeating the value everywhere? I would place it in `file_util.go` and name `ModeReadWrite`.